frontend/next-i18next.config.js

```diff
@@ -2,9 +2,10 @@
 
 module.exports = {
   i18n: {
-    defaultLocale: "en",
+    defaultLocale: "en_GB",
     locales: [
-      "en",
+      "en_GB",
+      "en_US",
```

Comment:
In my experience it is very unusual to have a specific US locale. More typically there is an 'en' one and then some variations like GB, CA, AU, ...

Reply:
I'm not sure why that would matter, I think the website will route you to the US version if your browser says you would like that.

But yeah, reordering would probably help us avoid the weblate breakage.